opcua.cpp

```diff
@@ -722,18 +722,28 @@ int n_subscriptions = 0;
 /**
  * Stop all subscriptions and disconnect from the OPCUA server
  */
-void
-OPCUA::stop()
+void OPCUA::stop()
 {
-	if (m_connected)
+	try
 	{
-		subscriptionVariables.clear();
-		m_assetPathNames.clear();
-		m_client->Disconnect();
+		if (m_connected)
+		{
+			subscriptionVariables.clear();
+			m_assetPathNames.clear();
+			m_client->Disconnect();			
+		}
+		if (m_client)
+		{
+			delete m_client;
```

Comment:
Set m_client back to NULL so that other tests will fail

Reply:
done